crates/store/src/db/sql.rs

```diff
@@ -483,17 +486,19 @@ pub fn select_notes_since_block_by_tag_and_sender(
         let note_type = row.get::<_, u8>(4)?;
         let sender = column_value_as_u64(row, 5)?;
         let tag: u32 = row.get(6)?;
-        let merkle_path_data = row.get_ref(7)?.as_blob()?;
+        let aux: u64 = row.get(7)?;
+        let aux = aux.try_into().map_err(|err| DatabaseError::DeserializationError(DeserializationError::InvalidValue(err)))?;
```

Comment:
I'm not sure this error represents the problem correctly here. Looking at the definition it seems that it's related to problems deserializing blobs where here, the `u64` was already deserialized from the database. But also, not sure if there is a better representation there. Maybe introducing a new `InvalidFelt` variant?

Reply:
done!